cmd/ddev/cmd/sequelpro.go

```diff
@@ -46,9 +45,7 @@ func handleSequelProCommand(appLocation string) (string, error) {
 		return "", err
 	}
 
-	nameContainer := fmt.Sprintf("%s-db", app.ContainerName())
-
-	if !dockerutil.IsRunning(nameContainer) {
+	if app.SiteStatus() != "running" {
```

Comment:
One of the new static analysis tools in https://github.com/drud/build-tools/pull/24 points out places where strings should be turned into symbols, so it will make sense to do it there.